roles/neutron-common/templates/etc/neutron/dhcp_agent.ini

```diff
@@ -6,6 +6,8 @@ debug = {{ neutron.logging.debug }}
 
 enable_isolated_metadata = True
```

Comment:
thats true we are now essentially forcing isolated metadata with this option, so I will remove this flag from our defaults